### PR TITLE
Info label with tooltip

### PR DESCRIPTION
**Pull request by elgatovital**
Changes proposed in this pull request:
- Add view for labels with info icon and tooltip
image

### CODE REVIEW

File: Multisig/UI/UI Library/Views/InfoLabel.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    
+    private var tooltipSource: TooltipSource?
+    
+    required init?(coder decoder: NSCoder) {
+        super.init(coder: decoder)
+    }
+    
+    func setText(_ text: String = "", description: String? = nil, style: GNOTextStyle = .headline) {
+        
+        let result = NSMutableAttributedString()
```
Comment (sche) [inline Multisig/UI/UI Library/Views/InfoLabel.swift]: No need for this variable. You can use `attributedText` var

**Review by sche — APPROVED** [review]

File: Multisig/Assets.xcassets/ico-info.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (sche) [inline Multisig/Assets.xcassets/ico-info.imageset/Contents.json:1]: You can use `info.circle` system icon

Comment (elgatovital) [reply Multisig/Assets.xcassets/ico-info.imageset/Contents.json:1]: it's a different icon with a different look. I would not use it. Instead, I have created dark mode version for icon from our design and added it to the project.

Comment (elgatovital) [reply Multisig/Assets.xcassets/ico-info.imageset/Contents.json:1]: image

**Pull request merged**